packages/twenty-website/src/app/api/releases/route.tsx

```diff
@@ -42,7 +43,10 @@ export async function GET() {
       };
     });
 
-    return Response.json(formattedReleasesNotes);
+    const response = NextResponse.json(formattedReleasesNotes);
+    response.headers.append('Access-Control-Allow-Origin', '*');
```

Comment:
For self hosted instances this is needed ... This shouldn't be a security problem?

Reply:
Would you mind updating middleware.js to allow `*` directly on twenty-website? I think it was unnecessary to limit it (cc @ady-beraud fyi)

Reply:
Sure done!